Mathlib/RingTheory/Kaehler.lean

```diff
@@ -545,7 +530,7 @@ theorem KaehlerDifferential.total_surjective :
   rw [← LinearMap.range_eq_top, Finsupp.range_total, KaehlerDifferential.span_range_derivation]
 #align kaehler_differential.total_surjective KaehlerDifferential.total_surjective
 
-set_option maxHeartbeats 3200000 in -- 2569452
+set_option maxHeartbeats 400000 in -- 2569452
```

Comment:
```suggestion
set_option maxHeartbeats 400000 in
```
I guess this is not relevant anymore.

Reply:
I've applied this change at https://github.com/leanprover-community/mathlib4/pull/6141